File: cases/taint/python/model-entrypoint-selectivity-positive/handler.py
def dfb_sink(value):  # DFB-SINK: model-entrypoint-selectivity-sink
    pass


def on_declared(payload):  # DFB-SOURCE: model-entrypoint-selectivity-input
    dfb_sink(payload)


def on_undeclared(payload):
    dfb_sink("clean")

File: cases/taint/java/model-entrypoint-selectivity-negative/Handler.java
package dataflowbench.taint;

final class Handler {
    static void dfb_sink(String value) { }  // DFB-SINK: model-entrypoint-selectivity-sink

    void onDeclared(String input) {
        dfb_sink(input);
    }

    void onUndeclared(String input) {  // DFB-SOURCE: model-entrypoint-selectivity-input
        dfb_sink(input);
    }
}
